File: code/dsa/linked_lists/python/creation_iterative.py
from .definition import Node


def create_linked_list(array):
    """Creates a linked list from a list of items"""
    head = Node(value=array[0])
    current = head
    if len(array) > 1:
        for val in array[1:]:
            next_item = Node(value=val)
            current.next = next_item
            current = next_item
    return head

File: source/code/dsa/linked_lists/javascript/creation_iterative.js
import { Node } from './definition.js';

export function createLinkedlist(array) {
    head = Node(array[0]);
    current = head
    if ( array.len() > 1 ) {
        for ( i=1; i<array.len(); i++  ) {
            item = array[i];
        }
    }
    return head
};
